src/Native/Runtime/gcrhenv.cpp

```diff
@@ -205,6 +205,7 @@ bool RedhawkGCInterface::InitializeSubsystems(GCType gcType)
     // Set the GC heap type.
     bool fUseServerGC = (gcType == GCType_Server);
     InitializeHeapType(fUseServerGC);
+    g_heap_type = fUseServerGC ? GC_HEAP_SVR : GC_HEAP_WKS;
```

Comment:
This should better be in `InitializeHeapType`

Reply:
`g_heap_type` is the runtime side field and `g_gc_heap_type` is the field on the GC side, InitializeHeapType is a function on the GC side, so this better stays on this side?

Reply:
I missed that these are two different variables. What you have here is fine. Thanks!